Bug: package runtime/trace is not in GOROOT

I am having this issue while trying to compile my program to -target wasm:

`tinygo build -o ./nodejs/lib-tiny.wasm -target wasm ./main.go`

It throws this error:
`C:\Users\**\go\pkg\mod\go.opencensus.io@v0.23.0\trace\trace_go11.go:21:2: package runtime/trace is not in GOROOT (C:\Users\**\AppData\Local\tinygo\goroot-go1.16.5-b6761f345c63a2cb198265aef5bcc7da1c2af2c14394faa25cfb608af5b42a9c\src\runtime\trace)`

Is there a way to print more build error stack trace info and any solution for the `package runtime/trace` issue?

Tinygo env:
![image](https://user-images.githubusercontent.com/13569609/125237712-f931d180-e307-11eb-84ea-af0c545ad72b.png)

Similar issues in another repo: https://github.com/benthosdev/benthos-lab/issues/22
Best.

--- a/src/runtime/pprof/pprof.go
+++ b/src/runtime/pprof/pprof.go
@@ -10,8 +10,7 @@ import (
 
 var ErrUnimplemented = errors.New("runtime/pprof: unimplemented")
 
-type Profile struct {
-}
+type Profile struct{}
 
 func StartCPUProfile(w io.Writer) error {
 	return nil
@@ -28,6 +27,18 @@ func Lookup(name string) *Profile {
 	return nil
 }
 
+func (p *Profile) Name() string {
+	return ""
+}
+
+func (p *Profile) Count() int {
+	return 0
+}
+
 func (p *Profile) WriteTo(w io.Writer, debug int) error {
 	return ErrUnimplemented
+}
+
+func Profiles() []*Profile {
+	return nil
 }

--- a/src/runtime/trace/trace.go
+++ b/src/runtime/trace/trace.go
@@ -1,0 +1,13 @@
+// Stubs for the runtime/trace package
+package trace
+
+import (
+	"errors"
+	"io"
+)
+
+func Start(w io.Writer) error {
+	return errors.New("not implemented")
+}
+
+func Stop() {}

